File: SDKs/amazonaws.com/ec2/2016-11-15/typescript/src/sdk/models/shared/assignprivateipaddressesresult.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class AssignPrivateIpAddressesResult extends SpeakeasyBase {
  @SpeakeasyMetadata()
  assignedIpv4Prefixes?: Record<string, any>;

  @SpeakeasyMetadata()
  assignedPrivateIpAddresses?: Record<string, any>;

  @SpeakeasyMetadata()
  networkInterfaceId?: Record<string, any>;
}

File: SDKs/amazonaws.com/ec2/2016-11-15/python/src/sdk/models/shared/assignprivateipaddressesresult.py
import dataclasses
from typing import Any,Optional


@dataclasses.dataclass
class AssignPrivateIPAddressesResult:
    assigned_ipv4_prefixes: Optional[dict[str, Any]] = dataclasses.field(default=None)
    assigned_private_ip_addresses: Optional[dict[str, Any]] = dataclasses.field(default=None)
    network_interface_id: Optional[dict[str, Any]] = dataclasses.field(default=None)
    
